client/jetpack-connect/index.js

```diff
@@ -165,6 +165,7 @@ export default function () {
 
 	page( '/jetpack/sso/:siteId?/:ssoNonce?', controller.sso, makeLayout, clientRender );
 	page( '/jetpack/sso/*', controller.sso, makeLayout, clientRender );
-	page( '/jetpack/new', controller.newSite, makeLayout, clientRender );
+	// Skip Jetpack modal
```

Comment:
Since the URL is "jetpack/new", it might be confusing later on to understand why this redirects to WordPress.com signup page, not to `jetpack/connect`.

So just elaborating that previously this route allowed creating both .com sites and connecting Jetpack sites would be good.

Reply:
Good point — I've added a more detailed explanation